resources/views/attributes/DateTimeAttribute.blade.php

```diff
@@ -1,16 +1,37 @@
-<label class="form-group js-placeholder" for="{{ $name }}">
+<style>
+    .datepicker input {
+        width: 40%;
+    }
+    .datepicker .btn {
+        border: 1px solid #dadada;
+        border-left: none;
+        font-size:1.12em;
```

Comment:
Since we use 16 as the base font size, please use 1.125em to get 18px (18 / 16 = 1.125).